src/styles/blocks/stars.css

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+
+  min-width: 96px;
+}
+
+.stars__star {
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+
+  width: 16px;
+  height: 16px;
+}
+
+.stars--4 > .stars__star:nth-child(-n + 4) {
+  background-image: url(/images/star-active.svg);
+}
```

Comment:
You miss styles for others stars. The product can have different rating, so this need to be flexible.

Reply:
Oh thanks, I'll add it